airbyte-integrations/connectors/source-twilio/integration_tests/expected_records.jsonl

```diff
@@ -1,63 +0,0 @@
-{"stream": "addresses", "data": {"sid": "ADa29b1ee20cf61d213f7d7f1a3298309a", "account_sid": "ACdade166c12e160e9ed0a6088226718fb", "friendly_name": null, "customer_name": "test-customer_name_1", "street": "test-street_1", "street_secondary": null, "city": "test-city_1", "region": "test-region_1", "postal_code": "test-postal_code_1", "iso_country": "US", "uri": "/2010-04-01/Accounts/ACdade166c12e160e9ed0a6088226718fb/Addresses/ADa29b1ee20cf61d213f7d7f1a3298309a.json", "date_created": "2020-11-25T09:41:48Z", "date_updated": "2020-11-25T09:41:48Z", "emergency_enabled": false, "validated": false, "verified": false}, "emitted_at": 1682602854052}
```

Comment:
You need to return this data.

Reply:
@marcosmarxm should I be returning the exact data? I've already added the stream to the configured_catalog right with the apporpriate schema. Or is there something else?

Reply:
Yes, this file is needed to run integration tests in our CI env.

Reply:
Also check unit tests are not working too.

Reply:
@marcosmarxm  okay I have pushed the change to add the records file.
As for the unit tests, how do I import the streams from the manifest.py file into the unit test test_source.py for the test to run? That seems to be the problem from stopping the unit test to be succesful